rubble/src/link/queue.rs

```diff
@@ -190,6 +190,21 @@ impl<T> Consume<T> {
             result,
         }
     }
+
+    /// Retrieves whether this consume represents consuming a packet or not.
+    pub fn consume(&self) -> bool {
```

Comment:
Maybe `should_consume` would be better? Feel free to rename the field to match as well.

Reply:
Renamed!